packages/fxa-settings/src/components/PageSettings/index.stories.tsx

```diff
@@ -26,6 +27,7 @@ storiesOf('Pages/Settings', module)
           recoveryKey: false,
           totp: { exists: false, verified: false },
           attachedClients: SERVICES_NON_MOBILE,
+          linkedAccounts: MOCK_LINKED_ACCOUNTS,
```

Comment:
Not necessary for this PR but it would also be super nice for this file to get reformatted to the new Storybook since it was touched 🙈 

I think maybe for "Cold start" story (this one), we might want to not show `linkedAccounts`. That story is supposed to be a "bare minimum" example if someone just created an account - I think in fact it would be nice for `attachedClients` to just show one session, but that again isn't part of the issue you fixed, so 🤷‍♀️

Reply:
Great thinking, will do!